### PR TITLE
PING Review of Mitigation Strategies

### PR DESCRIPTION
**Pull request by jasonanovak**
Proposed PING edits for the "Mitigation Strategies" section.

### CODE REVIEW

**Comment by jasonanovak** [comment]
From Wendy on the mailing list:

> In light of the conversation last week on device memory and JS v header
> invocation, is it worth considering a mitigation of the form "require
> usage to be observable or auditable"?
> 
> For example, even if a feature's activation or use gives no specific
> warning to the end-user, it might be observable by the user, by
> third-party tools operating on the user's behalf, or by
> observers/analysts doing web crawls or investigations at an ecosystem
> level. Features with anticipated privacy impacts should make their use
> detectable.

File: index.src.html [hunk 1]
```diff
@@ -610,87 +610,190 @@ <h2 id="mitigations">
...
-      system owners, or managers responsibility to realize this and act accordingly in the design of his/her
-      system. Some use may warrant conducting as privacy impact assessment, especially when data relating to
-      individuals may be processed. Examples.
+  <h3 id="privacy-impact-assessment">
+    Making a privacy impact assessment
+  </h3>
 
-      *   [GENERIC-SENSORS] advices to consider performing of a privacy impact assessment
+  Some features are potentially supplying very sensitive data, and it is
+  the end-developer, system owners, or managers responsibility to realize
```
Comment (wseltzer) [inline index.src.html]: Suggest rewording to "..the responsibility of the end-developer, system owner, or manager to realize..." to avoid making them all possessive

File: index.src.html [hunk 2]
```diff
@@ -610,87 +610,190 @@ <h2 id="mitigations">
...
+  this and act accordingly in the design of his/her system. Some use may
+  warrant conducting as privacy impact assessment, especially when data
```
Comment (wseltzer) [inline index.src.html]: s/as/a/

File: index.src.html [hunk 3]
```diff
@@ -610,87 +610,190 @@ <h2 id="mitigations">
...
+      only needs to access a specific field of a user address book, it should be
+      possible to explicitly mark that field in the API call so that the user
+      agent can inform the user that this single field of data will be shared.
+  *   APIs should make it possible for user agents to let the user select,
+      filter, and transform information before it is shared with the requester.
+      The user agent can then act as a broker for trusted data, and will only
+      transmit data to the requester that the user has explicitly allowed.
+
+  Data minimization is applicable to specification authors, implementers as
+  well as to those deploying the final service.
```
Comment (wseltzer) [inline index.src.html]: "is applicable to specification authors and implementers, as well as..."

File: index.src.html (hunk 1 above)
Comment (jasonanovak) [reply index.src.html]: Thanks! This is addressed in 444a3f0.

File: index.src.html (hunk 2 above)
Comment (jasonanovak) [reply index.src.html]: Thanks! This is addressed in 444a3f0.

File: index.src.html (hunk 3 above)
Comment (jasonanovak) [reply index.src.html]: Thanks! This is addressed in 444a3f0.
